x/validator-preference/valpref-module/module.go

```diff
@@ -0,0 +1,210 @@
+package validator_preference
```

Comment:
Why is this not in the root of the module?

Reply:
just wanted to follow twap folder structure